evaluate_tflite.py

```diff
@@ -41,15 +41,18 @@ def tflite_worker(model, alphabet, lm, trie, queue_in, queue_out, gpu_mask):
     while True:
         msg = queue_in.get()
 
-        fin = wave.open(msg['filename'], 'rb')
+        filename = msg['filename']
+        wavname = os.path.splitext(os.path.basename(filename))[0]
+        fin = wave.open(filename, 'rb')
         fs = fin.getframerate()
         audio = np.frombuffer(fin.readframes(fin.getnframes()), np.int16)
         audio_length = fin.getnframes() * (1/16000)
         fin.close()
     
         decoded = ds.stt(audio, fs)
         
-        queue_out.put({'prediction': decoded, 'ground_truth': msg['transcript']})
+        queue_out.put({'wav': wavname, 'prediction': decoded, 'ground_truth': msg['transcript']})
+        print(queue_out.qsize(), end='\r')
```

Comment:
debugging leftover?

Reply:
No, I intentionally print the size (at the same line with '\r') so that people know the current progress.

Reply:
ok